src/features/profile/ProfilePage.tsx

```diff
@@ -55,12 +58,111 @@ function ProfilePage() {
       )}
     >
       <div style={{ height: 2000, backgroundColor: '#f8fcff' }}>
-        <br />
-        <ActiveTab />
-        <br />
-        mode:
-        {' '}
-        {profileMode}
+        <ul className="profile-data">
+          <li>
+            <b>Имя:</b>
+            {' '}
+            {mockData.name}
+          </li>
+          <li>
+            <b>Фамилия:</b>
+            {' '}
+            {mockData.surname}
+          </li>
+          <li>
+            <b>Электронная почта:</b>
+            {' '}
+            {mockData.email}
+          </li>
+          <li>
+            <b>Ставка в час:</b>
+            {' '}
+            {mockData.ratePerHour}
+            {' '}
+            рублей
+          </li>
+          <li>
+            <b>Полный оклад:</b>
+            {' '}
+            {mockData.fullSalary}
+            {' '}
+            рублей
+          </li>
+          <li>
+            <b>Ставка:</b>
+            {' '}
+            {mockData.employmentType}
+          </li>
+          <li>
+            <b>Оклад:</b>
+            {' '}
+            {mockData.salary}
+            {' '}
+            рублей
+          </li>
+          <li>
+            <b>Фактическая стоимость часа:</b>
+            {' '}
+            {mockData.hourCostFact}
```

Comment:
for easy reading of the numbers, let's be thousands 
example: 100 000 
Will help you with you https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat